share/spice/just_delete_me/content.handlebars

```diff
@@ -0,0 +1,15 @@
+{{#if notes}}
+  <p>
+    {{notes}}
+  </p>
+{{/if}}
+{{#if delete_url}}
+  <p>
+    <b>URL:</b> <a href='{{delete_url}}'>{{delete_url}}</a>
+  </p>
+{{/if}}
+{{#if email}}
+  <p>
+    <b>Email:</b> <a href='mailto:{{email}}'>{{email}}</a>
```

Comment:
Yes, the email is provided from justdeleteme for requesting account deletion. I could possibly have fallback notes, one for email, other for url, though problem is what to do when there are both.